package-lock.json

```diff
@@ -4,6 +4,15 @@
   "lockfileVersion": 1,
   "requires": true,
   "dependencies": {
+    "JSONStream": {
```

Comment:
Weird you get these changes; I'm using node 8.5.0, and manually cleaned up the `node_modules` folder and did `npm i` to get a fresh `package-lock.json` file. Not a big deal, I just don't get why this happens.

Reply:
Yeah I know. I'm on 8.4.0 and 5.3.0. I've been encountering this on a lot of projects. It's weird.